### DISCUSSION TITLE
No auto CHARGE or auto DROP

### DISCUSSION THREAD

**Discussion by ibrubeer** [artisan-roaster-scope/artisan#1350] [General]
I was periodically not getting an auto DROP mark on the graph at the end of a roast.  It was bugging me, so I started playing with the Alarms thinking that was the problem.  Now I get an auto CHARGE mark as soon as I hit START in Artisan, and I get no auto DROP mark.  I think I have messed up the alarms.  I am using this on a Bullet V2.  Can somebody please help me with what the alarms are supposed to be for Charge and Drop?  I have the boxes checked in events for auto Charge and auto Drop.

![Screenshot 2023-12-16 10 31 07](https://github.com/artisan-roaster-scope/artisan/assets/44100300/5f7812fd-b93d-4ca8-be2f-1e92905aa64a)

![Screenshot 2023-12-16 10 31 58](https://github.com/artisan-roaster-scope/artisan/assets/44100300/3982ad82-1cc8-493d-8262-b148e62b3a89)

**Comment by roasterdave** [comment]
Your alarms have been changed from the defaults as set by Config>> Machine>> Aillio>> Bullet R1...

The preferred solution is to perform a Help>> Factory Reset then again run through Config>> Machine>> Aillio>> Bullet R1...
  However, it appears you have made other customizations, which will all be lost.  Instead, you can manually repair your alarms to match the machine defaults as shown below.

![image](https://github.com/artisan-roaster-scope/artisan/assets/30009312/fd4c15bd-0711-4de3-864a-cac8cda1e95a)

_Note: the screenshot applies to v2.8.4.  In v2.10.0 the "above" Condition is replaced with ">"._

**Comment by ibrubeer** [comment]
Thank you very much!  Yes, I would prefer to not test back to default, so
your image helps a ton.  Just out of curiosity, what exactly does the value
refer to in regards to Charge, and the 3xT1: State?  How does a person
interpret that?

**Reply by MAKOMO** [reply]
You have to ask Aillio for the exact semantic of what is returned by the machine as state here.

**Comment by ibrubeer** [comment]
I assumed that you guys knew what those fields were, but are you saying
that Aillio is the one who came up with those values?

Thank you for the assistance!

Dennis
